handler/metadata.go

```diff
@@ -0,0 +1,74 @@
+package handler
+
+import (
+	"net/http"
+
+	"github.com/ONSdigital/dp-population-types-api/config"
+	"github.com/ONSdigital/dp-population-types-api/contract"
+	"github.com/ONSdigital/dp-population-types-api/datastore"
+	"github.com/go-chi/chi/v5"
+	"github.com/pkg/errors"
+)
+
+type Metadata struct {
+	cfg         *config.Config
+	respond     responder
+	MongoClient Datastore
+}
+
+// NewMetada returns a new Metadata Handler
+func NewMetada(cfg *config.Config, r responder, databaseClient Datastore) *Metadata {
+	return &Metadata{
+		cfg:         cfg,
+		respond:     r,
+		MongoClient: databaseClient,
+	}
+}
+
+func (m *Metadata) PutMetadata(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+
+	populationType := chi.URLParam(r, "population-type")
+	var req contract.PutMetadataRequest
+	if err := parseRequest(r, &req); err != nil {
+		m.respond.Error(ctx, w, http.StatusNotFound, &Error{
+			err: errors.Wrap(err, "query parameter error"),
```

Comment:
If you're just using the `err` field of the error, you can return the error directly without using the Error struct